stubs/fpdf2/fpdf/table.pyi

```diff
@@ -34,7 +36,7 @@ class Table:
         headings_style: FontFace = ...,
         line_height: int | None = None,
         markdown: bool = False,
-        text_align: str | Align = "JUSTIFY",
+        text_align: str | _TextAlign | tuple[str | _TextAlign] = "JUSTIFY",
```

Comment:
```suggestion
        text_align: str | _TextAlign | tuple[str | _TextAlign, ...] = "JUSTIFY",
```
I assume it can't just be a single-element tuple

Reply:
Great find, common typo :)